examples/contracts/generics_forwarded/src/cw1.rs

```diff
@@ -0,0 +1,111 @@
+use cosmwasm_schema::schemars::JsonSchema;
+use cosmwasm_std::{CosmosMsg, CustomMsg, Response, StdError, StdResult};
+use cw1::{CanExecuteResp, Cw1};
+use serde::de::DeserializeOwned;
+use serde::Deserialize;
+use sylvia::contract;
+use sylvia::types::{CustomQuery, ExecCtx, QueryCtx, SvCustomMsg};
+
+#[contract(module = crate::contract)]
+#[messages(cw1 as Cw1)]
+#[sv::custom(msg=SvCustomMsg, query=SvCustomQuery)]
+impl<
+        InstantiateT,
+        Exec1T,
+        Exec2T,
+        Exec3T,
+        Query1T,
+        Query2T,
+        Query3T,
+        MigrateT,
+        CustomMsgT,
+        CustomQueryT,
+        FieldT,
+    > Cw1
+    for crate::contract::GenericsForwardedContract<
+        InstantiateT,
+        Exec1T,
+        Exec2T,
+        Exec3T,
+        Query1T,
+        Query2T,
+        Query3T,
+        MigrateT,
+        CustomMsgT,
+        CustomQueryT,
+        FieldT,
+    >
+where
+    for<'msg_de> InstantiateT: CustomMsg + Deserialize<'msg_de> + 'msg_de,
+    Exec1T: CustomMsg + DeserializeOwned + 'static,
```

Comment:
It seems like `cosmwasm_std::CustomMsg + DeserializeOwned` is the same as `sylvia::types::CustomMsg`, so those trait bounds can be simplified.

Reply:
As those examples are not only meant for the user but also to test our code generation in some places I used combined bounds to check if we properly forward those in the generated code. In most places though I tried to use the `SvCustomMsg` to simplify the examples.